tests/security/test_pausable.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+from starkware.starknet.testing.starknet import Starknet
+from utils import (
+    TRUE, FALSE, assert_revert, assert_event_emitted, 
+    get_contract_def, cached_contract, TestSigner
+)
+
+signer = TestSigner(12345678987654321)
+
+@pytest.fixture
+async def pausable_factory():
+    pausable_def = get_contract_def("tests/mocks/Pausable.cairo")
+    account_def = get_contract_def("openzeppelin/account/Account.cairo")
+
+    starknet = await Starknet.empty()
+    pausable = await starknet.deploy(
+        contract_def=pausable_def,
+        constructor_calldata=[]
+    )
+    account = await starknet.deploy(
+        contract_def=account_def,
+        constructor_calldata=[signer.public_key]
+    )
+    state = starknet.state.copy()
+
+    pausable = cached_contract(state, pausable_def, pausable)
+    account = cached_contract(state, account_def, account)
+    return pausable, account
+
+
+@pytest.mark.asyncio
+async def test_pausable_when_unpaused(pausable_factory):
+    contract, _ = pausable_factory
+
+    execution_info = await contract.isPaused().call()
+    assert execution_info.result.isPaused == FALSE
+
+    execution_info = await contract.getCount().call()
+    assert execution_info.result.res == 0
+    
+    # check that function executes when unpaused
+    await contract.normalProcess().invoke()
+
+    execution_info = await contract.getCount().call()
+    assert execution_info.result.res == 1
+
+    await assert_revert(
+        contract.drasticMeasure().invoke(),
+        reverted_with="Pausable: contract is not paused"
+    )
+
+@pytest.mark.asyncio
+async def test_pausable_when_paused(pausable_factory):
+    contract, _ = pausable_factory
+
+    execution_info = await contract.isPaused().call()
+    assert execution_info.result.isPaused == FALSE
+
+    # pause
+    await contract.pause().invoke()
+
+    execution_info = await contract.isPaused().call()
+    assert execution_info.result.isPaused == TRUE
+
+    await assert_revert(
+        contract.normalProcess().invoke(),
+        reverted_with="Pausable: contract is paused"
```

Comment:
```suggestion
        reverted_with="Pausable: paused"
```
to be consistent with solidity